style/servo/media_queries.rs

```diff
@@ -225,13 +254,20 @@ impl Device {
     /// when we get there
     pub fn query_font_metrics(
```

Comment:
This isn't possible yet, because the `FontMetrics` in Stylo (as opposed to the one in Servo -- which is confusing) does not expose `line_gap` and it isn't simply `ascent` + `descent`.

Reply:
That said, I'd love to tackle this in a quick followup.